tutorials/tools/Data_Simulator.ipynb

```diff
@@ -0,0 +1,347 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "\"\"\"\n",
+    "You can run either this notebook locally (if you have all the dependencies and a GPU) or on Google Colab.\n",
+    "\n",
+    "Instructions for setting up Colab are as follows:\n",
+    "1. Open a new Python 3 notebook.\n",
+    "2. Import this notebook from GitHub (File -> Upload Notebook -> \"GITHUB\" tab -> copy/paste GitHub URL)\n",
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "5. Restart the runtime (Runtime -> Restart Runtime) for any upgraded packages to take effect\n",
+    "\"\"\"\n",
+    "\n",
+    "## Install dependencies\n",
+    "!pip install wget\n",
+    "!apt-get install sox libsndfile1 ffmpeg\n",
+    "!pip install unidecode\n",
+    "!pip install matplotlib>=3.3.2\n",
+    "!pip install cmake\n",
+    "!pip install https://github.com/DavidDiazGuerra/gpuRIR/zipball/master\n",
+    "!pip install pyroomacoustics\n",
+    "\n",
+    "import json\n",
+    "import wget\n",
+    "import numpy as np\n",
+    "import os\n",
+    "\n",
+    "# option #1: local path to NeMo repo with NeMo already installed\n",
+    "NEMO_DIR_PATH = \"NeMo\"\n",
+    "\n",
+    "# option #2: download NeMo repo and install\n",
+    "if 'google.colab' in str(get_ipython()) or not os.path.exists(NEMO_DIR_PATH):\n",
+    "  ! git clone https://github.com/NVIDIA/NeMo\n",
+    "  %cd NeMo\n",
+    "  ! python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]\n",
```

Comment:
You are missing `BRANCH` variable for here.
Add a line `BRANCH=main`.
Also when you test, use `!pip install git+https://github.com/chooper1/NeMo.git@<commit_hash>` instead this line.

We have to assume that this code is merged to main, so please add `BRANCH = 'main'`.